picard/ui/widgets/multicombobox.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2019-2020, 2022 Philipp Wolfer
+# Copyright (C) 2020-2021, 2024 Laurent Monin
```

Comment:
This should be updated, I guess it was copied over.
Add your name, and current year and remove others (we didn't edit this file yet).